internal/perses/common/constants.go

```diff
@@ -16,8 +16,46 @@ limitations under the License.
 
 package common
 
+import "github.com/perses/perses-operator/api/v1alpha1"
+
 const (
 	PersesFinalizer     = "perses.dev/finalizer"
 	TypeAvailablePerses = "Available"
 	TypeDegradedPerses  = "Degraded"
+
+	// Volume names
+	ConfigVolumeName  = "config"
+	StorageVolumeName = "storage"
+
+	// TLS volume names
+	CAVolumeName     = "ca"
+	CAMountPath      = "/ca"
+	TLSVolumeName    = "tls"
+	TLSCertMountPath = "/tls"
+
+	// Mount paths
+	StorageMountPath  = "/etc/perses/storage"
+	ConfigMountPath   = "/perses/config"
```

Comment:
it's quite funny that in the Perses docker image, it's actually the opposite: 
- `/perses` is used for the storage
- `/etc/perses/config.yaml` is used to store the config

Maybe we should be aligned, WDYT @jgbernalp ?

Reply:
yes it makes sense. Will adjust.